vrc-get-gui/locales/ja.json5

```
@@ -0,0 +1,152 @@
+    "ready to create a project": "プロジェクトが作成可能です！",
+    "invalid project name": "プロジェクト名が不正です。",
+    "using such a symbol may cause problems": "そのような記号を使用すると問題が発生する可能性があります。",
+    "using multibyte characters may cause problems": "マルチバイト文字を使用すると問題が発生する可能性があります。",
+    "the directory already exists": "そのディレクトリは既に存在しています",
+    "checking the directory name...": "ディレクトリの名前を確認中...",
+    "template:": "テンプレート:",
+    "select directory": "ディレクトリを選択",
+    "the new project will be at <code>{{path}}</code>": "新しいプロジェクトは <code>{{path}}</code> に保存されます。",
+    "creating the project...": "プロジェクトを作成中...",
+    "create": "作成する",
+    "project created successfully": "プロジェクトを作成しました。",
+    "package": "パッケージ",
+    "installed": "インストール済み",
+    "latest": "最新のバージョン",
+    "source": "ソース",
+    "no upgradable packages": "更新可能なパッケージはありません。",
+    "installed {{name}} version {{version}}": "{{name}} バージョン {{version}} をインストールしました。",
+    "removed {{name}}": "{{name}} をアンインストールしました。",
+    "upgraded all packages": "すべてのパッケージを更新しました。",
+    "migrate project": "プロジェクトを移行",
+    "failed to migrate project: unity 2022 not found": "プロジェクト移行に失敗しました: Unity 2022がありません",
+    "failed to finalize the migration: unity 2022 not found": "プロジェクト移行の完了処理を行えませんでした: Unity 2022がありません",
+    "the project is migrated to unity 2022": "プロジェクトをUnity 2022に移行しました。",
+    "located at: <code>{{path}}</code>": "保存場所: <code>{{path}}</code>",
+    "unity version: ": "Unityバージョン: ",
+    "manage packages": "パッケージ管理",
+    "upgrade all": "すべて更新",
+    "reinstall all": "すべて入れ直す",
+    "select repositories": "VPMリポジトリを選択",
+    "official": "Official",
+    "curated": "Curated",
+    "unity 2019 to 2022 migration suggestion": "このプロジェクトは、VRChat SDKによるサポートが終了したUnity 2019を使用しています。VRChatはUnity 2022に移行することを推奨しています。",
+    "unity migration": "Unityバージョンを移行",
+    "we could not find unity exact recommended version of unity 2022": "推奨バージョンに一致するUnity 2022が見つかりませんでした。",
+    "recommended: {{version}}": "推奨バージョン: {{version}}",
+    "found: {{version}}": "見つかったバージョン: {{version}}",
+    "this may cause problems with VRChat SDK": "これにより、VRChat SDKで問題が発生する可能性があります。",
+    "do you want to continue?": "移行を続けますか？",
+    "continue": "続ける",
+    "please do not close the window": "ウィンドウを閉じないでください。",
+    "launching unity 2022 in background for finalizing the migration...": "Unity 2022をバックグラウンドで起動し、移行の完了処理を行っています...",
+    "you're applying the following changes to the project": "このプロジェクトに以下の変更を加えようとしています。",
+    "see changelog": "更新履歴を見る",
+    "apply changes": "変更を適用",
+    "install {{name}} version {{version}}": "{{name}} バージョン {{version}} をインストール",
+    "remove {{name}} since you requested": "{{name}} をアンインストール",
+    "remove {{name}} since it's a legacy package": "レガシーパッケージとして {{name}} をアンインストール",
+    "remove {{name}} since it's not used": "使用されていない {{name}} をアンインストール",
+    "there are version conflicts_one": "パッケージのバージョン指定が競合しています。",
+    "there are version conflicts_other": "パッケージのバージョン指定が競合しています。",
```

```suggestion
    "there are version conflicts": "パッケージのバージョン指定が競合しています。",
```

https://www.i18next.com/translation-function/plurals によって追加されたものなので日本語の場合分ける必要がないです